partner_invoicing_frequency/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+	:target: http://www.gnu.org/licenses/agpl
+	:alt: License: AGPL-3
+
+===================
+Invoicing Frecuency
+===================
+
+This module allows to assign a invoicing frecuency to contacts. This information is transferred to sale order, purchase orders, pickings and invoices, but it can be freely edited.
```

Comment:
```suggestion
This module allows to assign a invoicing frequency to contacts. This information is transferred to sale order, purchase orders, pickings and invoices, but it can be freely edited.
```